### ISSUE TITLE
MITRE ATT&CK Sub-technique support

### ISSUE THREAD

**Issue by thebleucheese** [SecurityRiskAdvisors/VECTR#70]
Note: We're currently working on MITRE ATT&CK Sub-technique support for the Test Case panel and the heat map now that sub-techniques are out of beta.  Keep an eye out for an updated VECTR version over the coming weeks.

**Comment by gregohmyeggo** [comment]
Hello @thebleucheese, 

Is there an ETA on support ATT&CK V7 for Sub-Techniques?

**Comment by thebleucheese** [comment]
Our tentative ETA is 2-5 weeks. The addition of sub-techniques introduced some pretty complex behavior when combined with our filters and showing / hiding results so we had to move some calculations from the front end to the back-end. 

We have it roughly working on a dev branch, but we've got a little bit more functionality to finish, some CSS tweaking, and compatibility checks across browsers.

Here's a screenshot of an earlier build  from late last week (the badge numbers are subject to change - we're still working on how we want to display total of parent technique + subtechniques vs just the count of the # of test cases in the parent)

![subtechniqueScreenshot](https://user-images.githubusercontent.com/30511683/90089577-b8909900-dcef-11ea-81f5-875e34f7e469.png)
